src/assets/data/defaultConfig.ts

```diff
@@ -143,3 +143,16 @@ export const DefaultRefreshTokenRequestBody =
 
 export const DefaultEnrolmentBaseURL =
   DefaultEndPoint + DefaultManagementPortalURI
+
+// TIME CONVERSIONS
+
+export const SEC_MILLISEC = 1000
```

Comment:
Would be good to add `TO` in these in between like `SEC_TO_MILLISEC`.

Reply:
I've created a function for them at #194 What do you think?

Reply:
Okay. I will have a look. Leave this be then